### ISSUE TITLE
API error format doesn't correspond to JSON API standard

### ISSUE THREAD

**Issue by servocoder** [servocoder/RichFilemanager#210]
The actual format is:
```
{
  "id": "server",
  "code": "500",
  "message": "Error details string",
  "arguments": {
    ... // list of error arguments
  }
};
```

However, according to the [JSON API standard](http://jsonapi.org/format/#errors) an RFM error response should looks as the following:
```
{
  "id": "server",
  "code": "500",
  "title": "Server error.",
  "detail": "Error details string",
  "meta": {
    "arguments": {
      ... // list of error arguments
    }
  }
};
```

**Comment by fabriceci** [comment]
The content of "_title_" should be "Error Code"  (because the connector send a code a not a title or a description).

The field "_detail_" is not useful in this plugin. (handled at the client side)

**Comment by servocoder** [comment]
Agree, then it will be:
```
{
  "id": "server",
  "code": "500",
  "title": "ERROR_CODE",
  "meta": {
    "arguments": {
      ... // list of error arguments
    }
  }
};
```

**Comment by servocoder** [comment]
Released in v2.7.0
